src/components/custTable.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+
+export class CustomersTable extends Component {
+  render() {
+    return (
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>First Name</th>
+            <th>Surname</th>
+            <th>Email</th>
+            <th>Room id</th>
+            <th>Check in date</th>
+            <th>Check out date</th>
+          </tr>
+        </thead>
+        <tbody id="rows">
+          <tr>
```

Comment:
There is a another pattern here 🙂 Would extracting a component be useful? This might also help with upcoming homework 😉